pkg/server/sandbox_run.go

```diff
@@ -180,10 +180,24 @@ func (c *criContainerdService) RunPodSandbox(ctx context.Context, r *runtime.Run
 			id, err)
 	}
 
-	// Add sandbox into sandbox store.
-	meta.CreatedAt = time.Now().UnixNano()
+	// Setup network.
 	// TODO(random-liu): [P2] Replace with permanent network namespace.
 	meta.NetNS = getNetworkNamespace(createResp.Pid)
+	if err = c.netPlugin.SetUpPod(meta.NetNS, config.GetMetadata().GetNamespace(), name, id); err != nil {
```

Comment:
That is because `docker create` doesn't really start the process. Actually docker start creates and starts the containerd container.

`create` in containerd/runc already starts the process, so we could initialize its network namespace.